source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -839,6 +843,32 @@ export class CardDesignerSurface {
         }
     }
 
+    // Outstanding issue: cannot drop an element into a ColumnSet
+    // Find the index of the designer surface where the dragged peer should be dropped
+    calculateDraggedPeerIndex(currentRoot: DesignerPeers.DesignerPeer): boolean {
+        // Need to be able to access the index of the dragged peer in the container
+        if (this._draggedPeer instanceof DesignerPeers.CardElementPeer) {
```

Comment:
what is CardElementPeer?

Reply:
`CardElementPeer` is the peer associated to a `CardElement` (to the best of my knowledge). Ensuring that is is a `CardElementPeer` lets us get the index of the `CardElement` in its container.

`draggedPeer` is the peer corresponding to the element that is being dragged around the card (either a new element or an existing element that the user is moving). We are trying to find the proper index to insert this peer at.